src/containers/bookform.js

```diff
@@ -0,0 +1,51 @@
+/* eslint-disable react/no-array-index-key */
+import React from 'react';
+
+class BooksForm extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      title: '',
+      category: '',
+    };
+    this.handleChange = this.handleChange.bind(this);
+    this.handleSelect = this.handleSelect.bind(this);
+  }
+
+  handleChange({ target }) {
+    this.setState({ title: target.value });
+  }
+
+  handleSelect({ target }) {
+    this.setState({ category: target.value });
+  }
```

Comment:
These can both be condensed into one function if the name of the target element is used as the setState object key

Reply:
Using the target as the name of the setState object key makes the input and the dropdown function in a strange way.

Reply:
If I may ask, strange way how?

**Note**: you will still need to condensed the two functions into one when you get to the next milestone which is one of the hard `requirement`
we should have `handleChange.bind` and `handleSubmit.bind`

Reply:
If I change the function to something like this:
```
handleInput({ target }) {
    this.setState({ target: target.value });
}
```
The input and the dropdown do not work. I tried to find out why but I couldn't.

Reply:
What if we use destructuring to get the target and set value. 🤔 
```
    handleChange(e) {
    const { target: { value, name } } = e;
    this.setState({
      [name]: value,
    });
  }
```

Reply:
Thanks so much for this.